src/radical/saga/adaptors/pbspro/pbsprojob.py

```diff
@@ -838,8 +835,6 @@ def _job_get_info(self, job_id, reconnect):
             if job_info.get('exec_host'):
                 job_info['exec_hosts'] = job_info['exec_hosts'].split('+')
 
-            # TORQUE doesn't allow us to distinguish DONE/FAILED on final state
-            # alone, we need to consider the exit_status.
```

Comment:
Not sure why the comment was removed?

Reply:
some cleanup, since it's for Torque, but this is PBSPro

Reply:
Ah, then never mind, it's a leftover then indeed...